tmt/utils.py

```diff
@@ -3044,15 +3049,15 @@ def _load(self, text: str) -> None:
                 "Unable to detect StructuredField version")
         self.version(int(version_match.groups()[0]))
         log.debug(
-            "Detected StructuredField version {0}".format(
+            "Detected StructuredField version {}".format(
```

Comment:
why are these not f-strings as all the others before?

apparently there are some more in convert, base and some here an there in other files as well

Reply:
My guess is that f-strings are supposed to be always used if the variable(s) are already assigned, while `.format` is ok when we're doing some operation.

Reply:
don't know how others feel about this, but IMO it would be great if we had all of them as f-strings and ditch the str.format, on the other hand in some places it might look bad/weird

Reply:
+1 for f-strings over `format()`. There's one exception, logging methods are allowed to use the printf-like formatting, `%d`, and so on, but my understanding was the f-strings were better & recommended everywhere else. And even if a computation is needed, it's either simple enough to fit into the f-string, or we can assign it to a helper variable first, for better readability, and use the f-string anyway.

Reply:
Just to clarify, it's not like I'm against using f-strings everywhere, but there currently isn't any ruff rule/settings that would allow us to check for that, at least as far as I know.  
I'm happy to work on it and get rid of the remaining `format()`s, but maybe it would be more suitable to separate PR?

Reply:
ok, I've added https://github.com/teemtee/tmt/pull/2014/commits/181d43eb09b165014f910a4bd453bef837e0df29 to hopefully get this resolved.  
I think it's not less (or more) readable than the str.format like this, though do agree helper variables could help with readability.